### ISSUE TITLE
wn.de

### ISSUE THREAD

**Issue by IDKwhattoputhere** [reek/anti-adblock-killer#1397]
What about paywalls?

`http://www.wn.de/`
After visiting 5 different articles:
![](https://cloud.githubusercontent.com/assets/16030872/15017042/4604d8f6-1214-11e6-8fb1-00799d67d136.png)
Chrome 50
Tampermonkey 4.0.25
uBlock Origin 1.7.2
Default + EasyList Germany + AakList

For example deleting cache and cookies does the job. Not sure if that's possible with AakScript or if there's a better solution.

**Comment by bourne9** [comment]
Filter - `wn.de###as-background`

**Comment by bourne9** [comment]
Try this - `wn.de###reduced_paywall_container`

**Comment by IDKwhattoputhere** [comment]
That just results in the message being hidden. The article is still only visible for the first part.

**Comment by bourne9** [comment]
It does what it is supposed to do, stop the message from loading, however, there is another layer of blurring element which is still loaded and needs to be blocked like the message too which I haven't been able to find it yet.

**Comment by reek** [comment]
@IDKwhattoputhere 
sorry but I don't kill paywalls

**Comment by bourne9** [comment]
Use BreakThePaywall chrome extension or simply don't allow wn.de to set any kind of cookies/data on to your device and it will never be able to detect how many articles you read and you will be a ghost.

**Comment by IDKwhattoputhere** [comment]
@reek I see. I guess it's not the purpose of this project anyways.

@bourne9 That's actually a good idea. Thanks!

**Comment by bourne9** [comment]
@IDKwhattoputhere and it works. I tested it right now. Simply, delete all your BROWSING DATA from chrome, then open content settings, open Cookies and Siet data, search for wn.de and if result appears, delete it and close it. then in Cookies section in Content settings, click Manage Exceptions and add wn.de and set it to Block and save it. You're done. Browse the website and read all articles you want, we just killed the triggering gun which stopped you from reading articles.
